src/Tokenizer/Token.php

```diff
@@ -216,6 +218,11 @@ public function equalsAny(array $others, bool $caseSensitive = true): bool
      */
     public static function isKeyCaseSensitive($caseSensitive, int $key): bool
```

Comment:
Why the method itself wasn't marked as `@deprecated`? It's better to have visual feedback when writing code instead of getting deprecation warning or tests failed when running it.

Reply:
#7601